src/modules/twap/state/twapOrdersListAtom.ts

```diff
@@ -1,64 +1,42 @@
 import { atom } from 'jotai'
 import { atomWithStorage } from 'jotai/utils'
 
-import { OrderStatus } from 'legacy/state/orders/actions'
-
-import { tokensByAddressAtom } from 'modules/tokensList/state/tokensListAtom'
 import { walletInfoAtom } from 'modules/wallet/api/state'
 
-import { OrderWithComposableCowInfo } from 'common/types'
 import { deepEqual } from 'utils/deepEqual'
 
-import { TwapOrderItem, TwapOrderStatus } from '../types'
-import { emulateTwapAsOrder } from '../utils/emulateTwapAsOrder'
+import { TwapOrderItem } from '../types'
 import { updateTwapOrdersList } from '../utils/updateTwapOrdersList'
 
 export type TwapOrdersList = { [key: string]: TwapOrderItem }
 
-export const twapOrdersListAtom = atomWithStorage<TwapOrdersList>('twap-orders-list:v4', {})
+export const twapOrdersAtom = atomWithStorage<TwapOrdersList>('twap-orders-list:v5', {})
```

Comment:
And this is why Elena's local storage is exploding :D

Reply:
Yeah, probably :)